TotalImage.IO/FileSystems/BPB/ExtendedBiosParameterBlock.cs

```diff
@@ -28,7 +28,7 @@ public override BiosParameterBlockVersion Version
             {
                 ExtendedBootSignature.Dos34 => BiosParameterBlockVersion.Dos34,
                 ExtendedBootSignature.Dos40 => BiosParameterBlockVersion.Dos40,
-                _ => throw new InvalidDataException()
+                _ => BiosParameterBlockVersion.Dos20,
```

Comment:
This might fix the crash, but it doesn't solve the problem that the EBPB class is being created for what obviously is not an EBPB.